### PR TITLE
HPCC-13678 Add Hungarian Translations

### PR DESCRIPTION
**Pull request by miguelvazq**
Add Hungarian Translations

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@AttilaVamos Please verify additions.

File: esp/src/eclwatch/nls/hu/hpcc.js [hunk 1]
```diff
@@ -207,6 +207,7 @@ define(
     IP: "IP",
     IPAddress: "IP cím",
     IsLibrary: "Könyvtár?",
+    IsReplicated: "Készült másodpéldány",
```
Comment (AttilaVamos) [inline esp/src/eclwatch/nls/hu/hpcc.js]: In this context (IsLibrary, IsReplicated) this message should close with question mark. "Készült másodpéldány?"

**Comment by miguelvazq** [comment]
@AttilaVamos added.

File: esp/src/eclwatch/nls/hu/hpcc.js [hunk 2]
```diff
@@ -605,6 +610,7 @@ define(
     ZeroLogicalFilesCheckFilter: "Nincs a megadott feltételnek megfelelő adat! (A jó a szűrőfeltétel?)",
     Zip: "Zip",
     ZippedAnalysisPackage: "Tömörített elemzési csomag",
+    Zoom: "Nagyítás",
     Zoom100Pct: "Nagyitás 100%-ra",
```
Comment (AttilaVamos) [inline esp/src/eclwatch/nls/hu/hpcc.js]: It seems I made a mistake in some previous translation. This should be "Nagyítás 100%-ra"

Comment (miguelvazq) [inline esp/src/eclwatch/nls/hu/hpcc.js]: There is no "%" in the new string we added "Zoom"

Comment (AttilaVamos) [inline esp/src/eclwatch/nls/hu/hpcc.js]: I mean not the new string is wrong. The next to it the Zoom100Pct should be "Nagyítás 100%-ra". The 'i' with different accent.

**Comment by miguelvazq** [comment]
Added.

**Comment by AttilaVamos** [comment]
Thanks, it is fine now.

**Pull request merged**